### ISSUE TITLE
Excluded apps are counting in Stats when Wintun is on

### ISSUE THREAD

**Issue by 2civics** [AdguardTeam/AdGuardVPNForWindows#607]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard VPN version

2.2.0 (1162)

### Operating mode

VPN

### Environment

```markdown
- OS: Windows 11 23H2 build 22631.2861
- Other network-level software:

BitDefender
Adguard for Windows
```

### Issue Details

Steps to reproduce:
1. Enable WinTun
2. Create exclusions for apps such as Fortinite.  Add all executables for Fortnite.
3. Play until you get disconnected (very high probability)
4. Check Stats in Adguard VPN.  You'll see that it is recording traffic routed through the VPN for the executables you excluded.
5. Disable WinTun
6. Clear Stats
7. Reboot
8. Play Fortnite again.  (You most likely will not get disconnected.)  I didn't
9. Check Stats.
10. Notice no traffic for exclusions being routed through Adguard VPN

### Expected Behavior

Expect any websites and/or executables added to the exclusions list to bypass the VPN.

### Actual Behavior

With WinTun enabled they seem to still be routed through the VPN.

### Screenshots

<details><summary>Screenshot 1:</summary>

</details>

**Comment by 2civics** [comment]
Just wanted to update that I've been gaming most of the day today with WinTun disabled and kept my Adguard VPN enabled and it worked flawlessly following the exlusions.

I've had nothing but problems with WinTun.  Adguard VPN crashing, not being able to connect after sleeping, and now WinTun not following exlusions.

Back to disabled and the VPN is rock solid.

**Comment by 2civics** [comment]
Another update since this seems to be completely ignored.

I have been testing alot.  So far I found that all exclusions seem to be working except for one Fortnite executable.  I don't understand why that one executable is going through the VPN when the WinTun is enabled.

**Comment by 2civics** [comment]
I don't know who or why the title got changed.  But the exclusions are being ignored.  It's not just stats being counted.  I use a seperate personal DNS server when passing through the VPN and another for the system.  

When using WinTun everything is hitting the DNS server setup for the VPN.   When I disable Wintun, my other personal server is getting use for stuff on the exlcusion list.

**Comment by KolbasovAnton** [comment]
@2civics Greetings! Bug is noticed, working on it. Most likely, it will be fixed in 2.2.1 version.

**Comment by 2civics** [comment]
Thank you.  I noticed you are the one that changed the title.  So when you say the bug is noticed are you referring to the bug of not following exclusions or what you changed the title to, which is just that stats are being counted?

**Comment by KolbasovAnton** [comment]
@2civics In this case, stats are illustrative example for incorrect filtering, of course we fixing the problem in general.

**Comment by 2civics** [comment]
Thanks.  Looking forward to the update.   WinTun seems like a great idea, but it's very buggy right now.  From this isssue to slower connections to not resuming from sleep, it's best disabled.  But I can't wait till all that's fixed and it works as intended. 

I've used your products for a very long time.   I have very much faith in Adguard's ability to fix this and make it amazing.

**Comment by 2civics** [comment]
Not fixed.  Nothing has changed in the update.

**Comment by 2civics** [comment]
How to reopen?   Behavior is exactly the same.  Nothing was ever fixed for this.

**Comment by KolbasovAnton** [comment]
@2civics Greetings!
Problem will be fixed in v2.2.2. We added the relevant tag in the labels (upper right corner).
